dissect/hypervisor/disk/vmdk.py

```diff
@@ -420,11 +475,15 @@ def parse(cls, vmdk_config):
                 continue
 
             if line.startswith("RW ") or line.startswith("RDONLY ") or line.startswith("NOACCESS "):
-                access_type, size, extent_type, filename = line.split(" ", 3)
-                filename = filename.strip('"')
-                size = int(size)
-                sectors += size
-                extents.append([access_type, size, extent_type, filename])
+                match = RE_EXTENT_DESCRIPTOR.search(line)
```

Comment:
The greediness of the `.+` operator seems to fix all side cases I can think of indeed. It also illustrates the issue I have with these regex parsers, they impose an unneeded cognitive load when trying to understand what they do exactly.

Reply:
> > What you'd want is a token parser which can deal with quotes, escaped quotes and such.
> 
> While that indeed sounds like a sane approach, this was not discussed in #45.

Well #45 was not a review, we had to revert because we couldn't release and as mentioned we only have limited resources, so in this case unfortunately there was no space to go deeper into the issue.